product_docs/docs/biganimal/release/getting_started/index.mdx

```diff
@@ -13,6 +13,8 @@ navigation:
 
 As a cloud administrator, you can set up BigAnimal with your existing Azure subscription or AWS account, invite others to join you in exploring what EDB has to offer, and create initial clusters as an account owner so that development can begin. 
 
+Alternatively, you can use [BigAnimal's Terraform provider](../using_cluster/terraform_provider) to incorporate BigAnimal into your existing cloud infrastructure workflows. *I'm not sure where in the process the Terraform provider comes in. Does mentioning it here make sense?*
```

Comment:
We should take this out for now. Won't be true until the next release. I'll try adding it as a comment, but there is a chance that will cause the build to fail. 

```suggestion
<!-- Alternatively, you can use [BigAnimal's Terraform provider](../using_cluster/terraform_provider) to incorporate BigAnimal into your existing cloud infrastructure workflows.  -->
```